README.md

```diff
@@ -228,6 +228,71 @@ To cleanup local environment do:
 make nuke
 ```
 
+## Apple Silicon Development
+
+Developing with apple silicon requires lima/colima installed on your machine. It sadly, did not work at all with kind.
+Below, you'll find the `yaml` used for developing with Cilium CNI and k3s.
+
+```yaml
+cpu: 6
+disk: 60
+memory: 16
+arch: host
+runtime: containerd
+kubernetes:
+  enabled: true
+  version: v1.28.2+k3s1
+  k3sArgs:
+    - --flannel-backend=none
+    - --disable=servicelb
+    - --disable=traefik
+    - --disable-network-policy
+    - --disable-kube-proxy
+autoActivate: true
+network:
+  address: false
+  dns: []
+  dnsHosts:
+    host.docker.internal: host.lima.internal
+  driver: slirp
+forwardAgent: false
+docker:
+  insecure-registries:
+    - localhost:5000
+    - host.docker.internal:5000
+vmType: vz
+rosetta: true
+mountType: virtiofs
+mountInotify: false
+cpuType: host
+layer: false
+provision:
+  - mode: system
+    script: |
+      set -e
+
+      # needed for cilium
+      mount bpffs -t bpf /sys/fs/bpf
+      mount --make-shared /sys/fs/bpf
+
+      mkdir -p /run/cilium/cgroupv2
+      mount -t cgroup2 none /run/cilium/cgroupv2
+      mount --make-shared /run/cilium/cgroupv2/
+      ln -s /opt/cni/bin/cilium-cni /usr/libexec/cni/cilium-cni
+sshConfig: true
+mounts: []
+env: {}
+cgroupsV2: false
+```
+
+### Steps
+
+1. In `Tiltfile.single`, ensure that you have commented out the `docker` portion and uncommented the `nerdctl` portion.
```

Comment:
what's the purpose of `Tiltfile.single`?

Reply:
why would't you just create a dedicated Tiltfile for lima VMs?

Reply:
perfect. so  there's one file for linux and one file for mac

Reply:
can you just update the readme and maybe rename the file to something reflecting that it's for mac or nerdctl.
otherwise single is a bit confusing.

Reply:
done, i opted for `Tiltfile.nerdctl` finally, it made more sense because nerdctl any os and the tiltfile isn't restricted to M powered macs